app/helpers/application_helper.rb

```diff
@@ -24,7 +24,7 @@ def current_path_without_query_string
   def show_breadcrumbs?(content_item)
     return false if content_item.nil?
 
-    no_breadcrumbs_for = %w[homepage landing_page service_manual_homepage service_manual_service_toolkit]
+    no_breadcrumbs_for = %w[flexible_page homepage landing_page service_manual_homepage service_manual_service_toolkit]
```

Comment:
Out of interest what does the breadcrumb look like if you do use application helper? I wonder if that's ok, rather than having to hard-code the breadcrumb. I'd imagine you'd want to make the basic structure as generic as possible.

Reply:
I think if you add a "parent" link to the content item, it will fix the breadcrumb issue.

Reply:
Reworked now.